config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
+  publishable_key: ENV['STRIPE_PUBLISHABLE_KEY'],
```

Comment:
Make sure to add these to `application.yml.template` file. It should have all the variables you need to setup.